### PR TITLE
Add ua.org.brezblock.q4wine

### PR DESCRIPTION
**Pull request by tim77**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/brezerk/q4wine/issues/147
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*
  - https://github.com/brezerk/q4wine/pull/194
  - https://github.com/brezerk/q4wine/pull/180

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

Potential maintainers: @brezerk, @gasinvein. Or @brezerk could take full control for sure over maintain if he wishes.

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build ua.org.brezblock.q4wine

**Comment by hfiguiere** [comment]
You need to disable `aarch64`

Also build fail because:

```
• url-not-found         : <screenshot> failed to download (HTTP 404: Not Found) [https://raw.githubusercontent.com/brezerk/q4wine/master/src/data/screenshots/q4wine-screenshot-1.png]
```

**Comment by tim77** [comment]
@hfiguiere as for screenshot we need to wait for upstream PR will be merged. Or figured out something else.

**Comment by tim77** [comment]
bot, build ua.org.brezblock.q4wine

File: flathub.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "skip-arches": ["aarch64"]
```
Comment (hfiguiere) [inline flathub.json:2]:
```suggestion
  "only-arches": ["x86_64"]
```

This is more inline with the intent

**Comment by barthalion** [comment]
Given the circumstances, I think it's safe to assume Brezerk may have bigger issues to handle than maintenance of this app. Would you be interested in co-maintanining it with tim77, @gasinvein?

**Comment by gasinvein** [comment]
@barthalion I'll help maintaining it if needed, but probably no need to invite me to collaborators since I'll have perms to the repo implicitly anyway.

**Pull request closed without merging**